libs/blocks/marquee/marquee.css

```
@@ -203,6 +203,20 @@
   object-position: left bottom;
 }
 
+.marquee .text .icon-area.icon-area-multiple {
+  display: flex;
+}
+
+.marquee .text .icon-area.icon-area-multiple img {
+  height: 40px;
+  max-height: 40px;
+  max-width: 114px;
+}
+
+.marquee .text .icon-area.icon-area-multiple > :not(:nth-child(1)) img {
```

This is basically saying the first child image then NOT the first child image. 

Can be simplified and should work as expected:

```suggestion
.marquee .icon-area-multiple :not(:first-child) img 
```